test/e2e/dra/utils/builder.go

```diff
@@ -44,11 +48,17 @@ import (
 	"k8s.io/utils/ptr"
 )
 
+var (
+	ExtendedResourceName = func(i int) string { return "example.com/gpu" + strconv.Itoa(i) }
```

Comment:
Why is this a variable and not a normal function?

Also, please document. It is not clear where the `"example.com/gpu"` string comes from and whether it has to be that  string (I think I know, but probably no-one else reading this will).

Reply:
changed to normal function, and added comment, esp about the hard coded string "example.com/gpu"